opengever/maintenance/browser/enable_sqlalchemy_logging_filter.py

```diff
@@ -0,0 +1,58 @@
+from five import grok
+from opengever.ogds.base.utils import create_session
+from Products.CMFPlone.interfaces import IPloneSiteRoot
+from threading import RLock
+import logging
+
+
+# Module globals to track whether the next bound SQL parameters should be
+# logged or not, and the according lock to ensure thread safety
+_log_next_param = False
+_log_next_param_lock = RLock()
```

Comment:
As discussed, this seems rather theoretical and should not really matter in practice. We'll leave the code as is for the moment :wink:.